src/Common/Core/Impl/Install/RInstallation.cs

```diff
@@ -123,33 +144,96 @@ public static class RInstallation {
             return path;
         }
 
-        public static void GoToRInstallPage() {
-            Process.Start("https://mran.revolutionanalytics.com/download");
+        public static Exception LaunchRClientSetup() {
+            var rClientExe = Path.Combine(Path.GetTempPath(), "RClientSetup.exe");
+            try {
+                using (var client = new WebClient()) {
+                    client.DownloadFileAsync(
+                        new Uri("http://go.microsoft.com/fwlink/?LinkId=800048", UriKind.Absolute), rClientExe);
+
+                    var startTime = DateTime.Now;
+                    while (client.IsBusy && (DateTime.Now - startTime).TotalMilliseconds < 5000) {
```

Comment:
Preventing them from using the UI might be prudent, but then you should use `RunSynchronouslyOnUIThread` - this will spin up the threaded wait dialog, so that OS doesn't treat VS as unresponsive.